README.md

```diff
@@ -36,6 +36,11 @@ Note: For actually enabling continuous integration for a GitHub hosted project,
     ```bash
     cabal new-install haskell-ci
     ```
+  or
+
+    ```bash
+    stack install --stack-yaml=stack-8.6.3.yaml
```

Comment:
can it be just `stack.yaml`?

*note* I **will** add a note that I (and neither Herbert) will maintain `stack.yaml`, so if this installation method bitrots, someone would need to step up and fix it.

Reply:
I can make the filename change.